q2_feature_table/_summarize/summarize_assets/webpack.config.js

```diff
@@ -0,0 +1,34 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    app: './src/main.js',
+    vendor: ['d3', 'javascript-natural-sort']
+  },
+  plugins: [
+    new webpack.optimize.CommonsChunkPlugin('vendor', 'app/vendor.bundle.js'),
+    new webpack.optimize.UglifyJsPlugin({
+      compress: { warnings: false }
```

Comment:
adding:
```javascript
mangle: {
    except: ['init']
}
```
to the uglifyjs config should allow you to use the JSONP directly instead of using single-letter globals to avoid the mangle (if I understand what's happening correctly).